tensorboard/webapp/widgets/line_chart_v2/lib/formatter_test.ts

```diff
@@ -101,41 +103,41 @@ describe('line_chart_v2/lib/formatter test', () => {
     }
   });
 
-  describe('#timeFormatter', () => {
+  describe('#wallTimeFormatter', () => {
     beforeEach(() => {
       TEST_ONLY.setLocale('en-US');
     });
 
     describe('#formatTick', () => {
       it('returns tick format using d3', () => {
         expect(
-          timeFormatter.formatTick(new Date('2020-1-5 13:23').getTime())
+          wallTimeFormatter.formatTick(new Date('2020-1-5 13:23').getTime())
         ).toBe('01:23');
         expect(
-          timeFormatter.formatTick(new Date('2020-1-5 5:00').getTime())
+          wallTimeFormatter.formatTick(new Date('2020-1-5 5:00').getTime())
         ).toBe('05 AM');
-        expect(timeFormatter.formatTick(new Date('2020-1-5').getTime())).toBe(
-          'Jan 05'
-        );
-        expect(timeFormatter.formatTick(new Date('2019-1-1').getTime())).toBe(
-          '2019'
-        );
+        expect(
+          wallTimeFormatter.formatTick(new Date('2020-1-5').getTime())
+        ).toBe('Jan 05');
+        expect(
+          wallTimeFormatter.formatTick(new Date('2019-1-1').getTime())
+        ).toBe('2019');
       });
     });
 
     describe('formatShort', () => {
       it('formats using localization', () => {
         expect(
-          timeFormatter.formatShort(new Date('2020-1-5 13:23').getTime())
-        ).toBe('1/5/20, 1:23:00 PM');
+          wallTimeFormatter.formatShort(new Date('2020-1-5 13:23').getTime())
+        ).toBe('Jan 5, 20, 1:23:00 PM');
```

Comment:
“Jan 5, 20” feels unfamiliar to me. I would probably expect that to mean
“the two days January 5 and January 20” rather than “January 5, 2020”.
Is “Jan 5, 2020” too long? What if we only include the year when it’s
not the current year? (just providing some ideas; not blocking)